internal/utils.py

```diff
@@ -1,91 +1,160 @@
+    Args:
+        command (_str_): Command to be executed.
+
+    Returns:
+        _tuple_: str, str
+    """
     try:
         test_type = os.getenv("TEST_TYPE") if os.getenv("TEST_TYPE") else None
         # getting command type
         sub_commands = command.split()
         cmd_type = None
         if len(sub_commands) > 1 and (
-            sub_commands[1] == "q" or sub_commands[1] == "query"
+            sub_commands[1] == "q"  # pylint: disable=C0330
+            or sub_commands[1] == "query"  # pylint: disable=C0330
         ):
             cmd_type = QUERY_TYPE
         elif len(sub_commands) > 1 and (sub_commands[1] == "tx"):
             cmd_type = TX_TYPE
 
-        stdout, stderr = subprocess.Popen(
+        stdout, stderr = subprocess.Popen(  # pylint: disable=R1732
             command.split(), stdout=subprocess.PIPE, stderr=subprocess.PIPE
         ).communicate()
-        out, err = stdout.strip().decode(), stderr.strip().decode()
+        out, error = stdout.strip().decode(), stderr.strip().decode()
         if test_type and cmd_type:
-            record_stat(test_type, cmd_type, out, err)
-        return out, err
-    except Exception as e:
+            record_stat(test_type, cmd_type, out, error)
+        if len(error) != 0:
+            return False, error
+        json_out = json.loads(out)
+        if cmd_type == TX_TYPE and "code" in json_out and json_out["code"] != 0:
+            return False, json_out
+        return True, json_out
+    except Exception as error:  # pylint: disable=W0703
         if test_type and cmd_type:
-            record_stat(test_type, cmd_type, "", e)
-        return None, e
+            record_stat(test_type, cmd_type, "", error)
+        return False, error
 
 
-# The utility function `is_tool` is used to verify the package or binary installation.
 def is_tool(binary):
-    """Check whether `name` is on PATH and marked as executable."""
-    from shutil import which
-
+    """
+    The utility function `is_tool` is used to verify the package or binary installation.
+    """
     return which(binary) is not None
 
 
-# validate_num_txs will validate num_txs value
-def validate_num_txs(x):
-    if int(x) < 1:
+def validate_num_txs(num_x):
+    """
+    validate_num_txs will validate num_txs value
+    Args:
+        num_x (_int_)
+
+    Raises:
+        argparse.ArgumentTypeError
+
+    Returns:
+        _int_: _int_
+    """
+    if int(num_x) < 1:
         raise argparse.ArgumentTypeError(
             "The argument NUM_TXS should be positive integer"
         )
-    return int(x)
+    return int(num_x)
+
+
+def num_txs_type(num_x):
+    """
+    num_txs_type is a user-defined type.
+    Args:
+        num_x (_unit_): int
+
+    Raises:
+        argparse.ArgumentTypeError
+
+    Returns:
+        _int_: int
+    """
+    if int(num_x) < 1000:
+        raise argparse.ArgumentTypeError("The argument NUM_TXS should be 1000 or more")
+    return int(num_x)
+
+
+def node_type(num_x):
```

Comment:
We don't need this. Let's allow even single node network

Reply:
Yeah, seems like we are not importing this function anywhere. Let's remove this function.